Bottom: 156ceaa72450e79337ebe107759691cbeeb9a927
Top:    79575a17402f01079802f50a15415ca4f986491d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-24 19:05:24 +0000

Refresh of add-routines-to-render-just-a-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 965f2b7..7d4e9a5 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -650,6 +650,7 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box, double scale
 
   glPopAttrib ();                               /* Restore the colour and stencil buffer write-mask etc.. */
   glPushAttrib (GL_STENCIL_BUFFER_BIT);         /* Save the stencil op and function */
+  glEnable (GL_STENCIL_TEST);                   /* Enable the stencil test, just in case it wasn't already on */
 
   glStencilOp (GL_KEEP, GL_KEEP, GL_INVERT);    /* This allows us to toggle the bit on the subcompositing bitplane */
                                                 /* If the stencil test has passed, we know that bit is 0, so we're */
